src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/form/main.js

```diff
@@ -411,16 +408,6 @@ define(['app-config', 'config', 'services/sulupreview/preview'], function(AppCon
             return url;
         },
 
-        showState: function(published) {
```

Comment:
This method handles the appaerance of the test or published icon in the toolbar, but was only implemented in the form. But it should have effect on all tabs, so I moved it.